contributors/design-proposals/volume-provisioning.md

```diff
@@ -412,9 +419,13 @@ type StorageClass struct {
   // Parameters for dynamic volume provisioner.
   Parameters map[string]string `json:"parameters,omitempty"`
 }
-
 ```
 
+StorageClass can have annotation
+`volume.beta.kubernetes.io/storage-class-selectors` that contains free-form
+english text with description of labels that are supported by the provisioner
+in pvc.spec.selector when provisioning a new PV for this StorageClass.
```

Comment:
As someone who is going to have to build UI on top of this, I would much rather see structured data here.

The original annotation proposal has the same translation concern you have here, unless you are proposing having a separate annotation for each language?